plot/index.html

```diff
@@ -1,7 +1,8 @@
 <html id="plot">
 
 <head>
-  <script src='../../chaise/chaise-config.js'></script>
+  <title>Plot App</title>
+  <script src='/chaise/chaise-config.js'></script>
```

Comment:
I'm not sure this should be changed. This will always use the `chaise-config` that's at the server root (the one used by master). This is fine for deployments using master (dev, staging, production) but makes it harder for us to test chaise config features and changes because our config will be ignored.